dockerfiles/control/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:16.04
+
+MAINTAINER ClusterHQ <contact@clusterhq.com>
+
+RUN \
+    apt-get --yes update \
+    && apt-get --yes install --no-install-recommends \
+        apt-transport-https \
+        ca-certificates \
+    && apt-get --yes clean
+
+ARG FLOCKER_VERSION="*"
+ARG FLOCKER_REPOSITORY="https://clusterhq-archive.s3.amazonaws.com/ubuntu/16.04/amd64"
+RUN echo "deb ${FLOCKER_REPOSITORY} /" > /etc/apt/sources.list.d/clusterhq.list
+RUN \
+    apt-get --yes update \
+    && apt-get --yes install --no-install-recommends --allow-unauthenticated \
+        clusterhq-python-flocker=${FLOCKER_VERSION} \
+        clusterhq-flocker-node=${FLOCKER_VERSION}
+
+EXPOSE 4523
+EXPOSE 4524
```

Comment:
a... right, mapping and exposing ports and mounting volumes is done in CLI. The reason we can add this in the Dockerfile is for declaration purpose. So we probably want to do that for both images.